plugin/src/main/java/org/opensearch/ml/breaker/MemoryCircuitBreaker.java

```diff
@@ -50,6 +50,6 @@ public Short getThreshold() {
 
     @Override
     public boolean isOpen() {
-        return jvmService.stats().getMem().getHeapUsedPercent() > this.getThreshold();
+        return getThreshold() < 100 && jvmService.stats().getMem().getHeapUsedPercent() > getThreshold();
```

Comment:
For better readability, we should put this `100` in a static variable and add some comment accordingly.